src/Repositories/VerificationCodesRepository.php

```diff
@@ -43,12 +43,11 @@ public function verifyCode(Verifiable $verifiable, string $action, string $code)
 
     private function updateVerifiedCode(VerificationCode $verificationCode, string $action, \DateTime $now): bool
     {
-        $actionVerifiedMinutes = !Config::get('verifications.actions.'. $action .'.keep_verified_during_session')
-                                ? Config::get('verifications.actions.'. $action .'.verified_action_valid_minutes')
-                                : null;
+        $actionVerifiedMinutes = Config::get('verifications.actions.'. $action .'.expires_in');
 
-        $verificationCode->verifies_until = $actionVerifiedMinutes ? Carbon::parse($now)->addMinutes($actionVerifiedMinutes)->toDateTime() : null;
+        $verificationCode->verifies_until = Carbon::parse($now)->addMinutes($actionVerifiedMinutes)->toDateTime();
```

Comment:
Je toto dobre? Pri kazdej verifikacii (nie len pri submitnuti kodu) sa ti aktualizuje verifies_until ? To asi nie je to ci chceme v pripade, ze expires_from == 'verification', nie? @miroslavtrnavsky @odziomkova-brackets ?

Reply:
Aha pockat, ta metodka `verifyCode` sa vola len raz pri pouziti kodu? Ak ano, tak potom ok. Ale co sa vola pri tom middlewari?

Reply:
Toto sa vykona len raz - tak ako pises, pri samotnej verifikacii kodu. Cize **verifies_until** sa nastavi prvy krat na pociatocnu hodnotu - pretoze toto je prvy "touch" od verifikacie, tym ze si ju uspesne vykonal. V middleware sa v pripade **expires_from == 'last-activity'** jednoducho updatuje tento parameter (cez verification facadu vo VerifiableTraite). Cize zjednodusene, zavola sa to len raz a nastavi to pociatocnu hodnotu.

Reply:
Daj link na riadok, kde sa to updatuje. Nevidim to nikde.